dev-infra/utils/BUILD.bazel

```
@@ -0,0 +1,16 @@
+load("@npm_bazel_typescript//:index.bzl", "ts_library")
+
+ts_library(
+    name = "config",
+    srcs = [
+        "config.ts",
+    ],
+    visibility = ["//dev-infra:__subpackages__"],
+    deps = [
+        "@npm//@types/node",
+        "@npm//@types/shelljs",
+        "@npm//comment-json",
```

This now needs to be switched to `json5` (and `@types/json5`?) too.

```suggestion
        "@npm//@types/json5",
        "@npm//@types/node",
        "@npm//@types/shelljs",
        "@npm//json5",
```